server/application/application.go

```diff
@@ -92,16 +92,9 @@ func appRBACName(app appv1.Application) string {
 	return fmt.Sprintf("%s/%s", app.Spec.GetProject(), app.Name)
 }
 
-func validatedSelector(selector string) string {
```

Comment:
this prevent defaulting to listing all apps when the label is bad

Reply:
this is bad, but it breaks UI. If user enter an invalid selector he don't have a chance to fix it any more.